fhir-model/src/main/java/com/ibm/fhir/model/builder/AbstractBuilder.java

```diff
@@ -1,12 +1,22 @@
 /*
- * (C) Copyright IBM Corp. 2019
+ * (C) Copyright IBM Corp. 2019, 2021
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
 package com.ibm.fhir.model.builder;
 
 public abstract class AbstractBuilder<T> implements Builder<T> {
+    protected boolean validating = true;
+
+    public void setValidating(boolean validating) {
+        this.validating = validating;
+    }
```

Comment:
I think we'll need to update our example generator to ensure that it skips these new public methods on the builders

Reply:
Fixed issue found in examples generator